.github/workflows/tests.yml

```diff
@@ -64,3 +64,21 @@ jobs:
       - name: Run tests
         run: |
           pytest vetiver -m 'rsc_test'
+
+  test-no-torch:
+    name: "Test no-torch"
+    runs-on: ubuntu-latest
+    if: ${{ !github.event.pull_request.head.repo.fork }}
```

Comment:
I think you can remove this line (I think it was copied from the RSConnect test job, which needs to skip if submitted from a fork). 

(You'll need you to use a branch on the main vetiver repo to run RS Connect tests)